Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:6.10.3-alpine
+
+WORKDIR /app
+
+COPY package.json /app/package.json
```

Comment:
You could consolidate the 3 separate COPYs into:
`COPY package.json package.config.js webpack.config.js /app/`
Optional, not a blocker.

Reply:
Yes.  Good call.